google-apps-script.js

```diff
@@ -59,7 +59,8 @@ function doPost(e) {
     if (sendEmailTo) {
       MailApp.sendEmail({
         to: String(sendEmailTo),
-        subject: "Contact form submitted",
+        name: String(e.parameters.formGoogleEmailSender) || "Contact Form",
+        subject: String(e.parameters.formGoogleEmailSubject) || "Contact form submitted",
```

Comment:
I believe these `|| ...` checks won't get hit in all cases.

If the parameters are null or undefined, (like if someone just updates the backend script here or doesn't give these parameters), then you would end up with `String(undefined) = 'undefined'` which is not falsey. 

It may be worth setting the name / subject above and consider using `mailData` and/or `.toString()` when the field is present/truthy.